tests/playwright-tests/README.md

```diff
@@ -0,0 +1,45 @@
+**How to install Playwright and set up VS Code?**
```

Comment:
Moving forwards, we'll also want to incorporate our other widely-used IDEs in here too e.g. Rider. But someone who uses them mostly can pick that part up :slightly_smiling_face:

Reply:
I managed to set it up in Pycharm . 
Happy to update the Readme as a separate PR after this is merged.
However I could not yet setup a Playwright test runner.